python/ShapeTools.py

```diff
@@ -114,6 +168,9 @@ def doIndividualModels(self):
                 self.out._import(sum_s, ROOT.RooFit.RenameConflictNodes(b))
                 if not self.options.noBOnly:
                     self.out._import(sum_b, ROOT.RooFit.RecycleConflictNodes(), ROOT.RooFit.Silence())
+            if self.options.newHist >= 1:
```

Comment:
Add the HistFuncWrappers to the workspace so we can extract individual process shapes scaled for the bin-by-bin uncertainties if we want to